docker/docker-compose.yml

```
@@ -0,0 +1,66 @@
+version: "3.6"
+
+services:
+  oeq:
+    build: .
+    networks:
+      - oeq-cluster
+    links:
+      - database
+    labels:
+      - "traefik.enable=true"
+      - "traefik.backend=oeq"
+      - "traefik.docker.network=oeq-cluster"
+      - "traefik.frontend.rule=Host:oeq.localhost"
+      - "traefik.port=8080"
+      - "traefik.backend.loadbalancer.stickiness=true"
+      - "traefik.backend.loadbalancer.stickiness.cookieName=JSESSIONNODE"
+      - "traefik.frontend.passHostHeader=true"
+    environment:
+      - EQ_MESSAGING_BINDPORT=8999
+      - EQ_MESSAGING_USEHOSTNAME=true
+      - EQ_USERSERVICE_USEXFORWARDEDFOR=true
+      - EQ_ZOOKEEPER_INSTANCES=zookeeper
+      - EQ_ZOOKEEPER_PREFIX=eqdev
+    depends_on:
+      - zookeeper
+    volumes:
+      - /home/penghai/Edalex/Equella/Dev/learningedge-config/data/filestore:/home/equella/equella/filestore
+      - /home/penghai/Edalex/Equella/Dev/learningedge-config/hikari.properties:/home/equella/equella/learningedge-config/hikari.properties
+      - /home/penghai/Edalex/Equella/Dev/learningedge-config/learningedge-log4j.properties:/home/equella/equella/learningedge-config/learningedge-log4j.properties
+
+  zookeeper:
+    image: zookeeper:latest
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_SERVERS: server.1=0.0.0.0:2888:3888;2181
+      ZOO_LOG4J_PROP: INFO, CONSOLE, ROLLINGFILE
+      ZOO_AUTOPURGE_PURGEINTERVAL: 1
+    volumes:
+      - /zookeeper/zoo1/data:/data
+      - /zookeeper/zoo1/datalog:/datalog
+    networks:
+      - oeq-cluster
+
+  database:
+    image: postgres
```

Again, probably best to pin to an exact version.

```suggestion
    image: postgres:9.6
```